Test/AwsCommonRuntimeKitTests/io/TLSContextTests.swift

```diff
@@ -4,26 +4,45 @@ import XCTest
 @testable import AwsCommonRuntimeKit
 
 class TLSContextTests: XCBaseTestCase {
+    
+    func testCreateTlsContextWithOptions() throws {
+        let options = TLSContextOptions()
+        let context = try TLSContext(options: options, mode: .client)
+        _ = TLSConnectionOptions(context: context)
+    }
+    
+#if os(macOS) || os(Linux)
+    func testCreateTlsContextWithFilePath() throws{
+        try skipIfiOS()
+        try skipIftvOS()
+        try skipIfwatchOS()
+        
+        let cert_path = try getEnvironmentVarOrSkipTest(environmentVarName: "AWS_TEST_MQTT311_IOT_CORE_X509_CERT")
+        let private_key_path = try getEnvironmentVarOrSkipTest(environmentVarName: "AWS_TEST_MQTT311_IOT_CORE_X509_KEY")
```

Comment:
I think we should keep variable naming consistent to camelCase. Underscore naming makes it seem like this has something to do with C.